### DISCUSSION TITLE
cue fastn?

### DISCUSSION THREAD

**Discussion by amitu** [fastn-stack/fastn#1283] [Ideas & RFCs]
https://news.ycombinator.com/item?id=37427722

> Basically, the idea is that users can register their project, tests, or benchmarks in Unity. The project maintainers can then test new code against these projects before releasing, or even before pushing a commit, because a regression is found for example.

**Comment by Arpita-Jaiswal** [comment]
We will create the fastn.com/cue.ftd which will list all the fastn projects that needs to be build successfully in order to create a new fastn release.

```ftd
-- fastn.cue: fastn.com
github-repo: https://github.com/fastn-stack/fastn.com

-- fastn.cue: fifthtry.com
github-repo: https://github.com/fastn-stack/fastn.com
```

Release will have a flag to do either quick release or normal release. Quick release will ignore the listed projects. Normal release makes sure that release will not break anything i.e. release doesn't have backward incompatible changes.

If test fails/passes then post a comment in the latest commit for which the test has run.

**Comment by amitu** [comment]
Svelte also has something like this: https://github.com/sveltejs/svelte-ecosystem-ci

**Comment by amitu** [comment]
We are creating issues with links to commits that are failing but should not. Soon we will have a lot of such issues. We should write a script which identifies all such issues (maybe using a label), marked closed, and ensure those commits are also still compiling before creating each release, else it is a regression.
